src/wallet.ts

```diff
@@ -1148,23 +1148,17 @@ export class Wallet {
         metaId: string
         storeId: string
         memo: string
-        tokens: {
-          royaltyPercent: string
-          royaltys: { account: string; percent: string }[]
-        }[]
+        royalties_percent: string
+        royalties: { account: string; percent: string }[]
```

Comment:
I think the `royalties` type is now `Record<string, number> `

Reply:
Done